src/qt/listbox.cpp

```diff
@@ -106,14 +106,17 @@ bool wxListBox::Create(wxWindow *parent, wxWindowID id,
 bool wxListBox::Create(wxWindow *parent, wxWindowID id,
             const wxPoint& pos,
             const wxSize& size,
-            const wxArrayString& WXUNUSED(choices),
+            const wxArrayString& choices,
             long style,
             const wxValidator& validator,
             const wxString& name)
 {
     Init();
     m_qtWindow = m_qtListWidget = new wxQtListWidget( parent, this );
 
+    for (size_t i = 0; i < choices.size(); ++i)
```

Comment:
I don't mind doing it like this, but I think it could be more efficient to construct a `QStringList` and call `addItems()` once, instead of calling `addItem()` in a loop many times.

Reply:
An excellent suggestion, I'll get this applied and committed ASAP. Many thanks!

Reply:
I am struggling to see a way I can do this change without having to loop over the `choices` array in some fashion. I am unable to convert the `wxString`s to `QString`s outside of a loop, so to apply your suggested change in this instance would mean we still have the same loop (albeit with different contents). Perhaps the string conversion in the loop along with the call to `addLines()` outside the loop would be more efficient but I am not certain is a worthy enough saving to warrant a further change.

Reply:
Sorry for being unclear, there will still definitely be a loop. The idea is just it's often more efficient to add many items to the control at once rather than adding them in a loop because you only pay the "item added" overhead (a control typically needs to do something more than just adding the string to an internal list) once instead of N times in this case. So "loop to create QStringList" + "a single call to addItems()" should be more efficient, especially for big amounts (and there can be hundreds of them in some listboxes), than "loop calling addItem".

Reply:
Excellent, I appreciate the clarification. I'll get this change committed soon!